### PR TITLE
Update IMPORT and IMPORT INTO permissions

### PR DESCRIPTION
**Pull request by shermanCRL**
Addresses https://github.com/cockroachdb/docs/issues/9179

### CODE REVIEW

File: v20.2/import-into.md [hunk 1]
```diff
@@ -4,18 +4,35 @@ summary: Import CSV data into an existing CockroachDB table.
...
-The `IMPORT INTO` [statement](sql-statements.html) imports CSV or Avro data into an [existing table](create-table.html). `IMPORT INTO` appends new rows onto the table.
+The `IMPORT INTO` [statement](sql-statements.html) imports CSV or Avro data into an [existing table](create-table.html), by appending new rows into the table.
 
 ## Considerations
 
-- `IMPORT INTO` only works for existing tables. For information on how to import data into new tables, see [`IMPORT`](import.html).
+- `IMPORT INTO` only works for existing tables. To import data into new tables, see [`IMPORT`](import.html).
 - `IMPORT INTO` cannot be used within a [transaction](transactions.html) or during a [rolling upgrade](upgrade-cockroach-version.html).
 - `IMPORT INTO` invalidates all [foreign keys](foreign-key.html) on the target table. To validate the foreign key(s), use the [`VALIDATE CONSTRAINT`](validate-constraint.html) statement.
 - `IMPORT INTO` cannot be used to insert data into a column for an existing row. To do this, use [`INSERT`](insert.html).
```
Comment (shermanCRL) [inline v20.2/import-into.md:14]: I’m not sure what this line means, do we know what it’s meant to convey?

**Review by lnhsingh — COMMENTED** [review]
nits on changing SPECIFIED / IMPLICIT to `SPECIFIED` and `IMPLICIT` on all versions / occurrences, but content :lgtm: otherwise.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9181#-:-MOcJZcjEnxFpdauMk2G:b-9z44ve)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @lnhsingh, @mwang1026, and @shermanCRL)

___
*[v20.2/import.md, line 34 at r3](https://reviewable.io/reviews/cockroachdb/docs/9181#-MOcGWnv6JRcIRJgi_3b:-MOcGWnv6JRcIRJgi_3c:b-4z9c0u) ([raw file](https://github.com/cockroachdb/docs/blob/39e1969e9c1134c0539d788e31276002ed6850a5/v20.2/import.md#L34)):*
> ```Markdown
> The source file URL does _not_ require the `ADMIN` role in the following scenarios:
> 
> - S3 and GS using SPECIFIED (and not IMPLICIT) credentials. Azure is always SPECIFIED by default.
> ```

nit: make SPECIFIED/IMPLICIT > `SPECIFIED` / `IMPLICIT`
___
*[v20.2/import.md, line 39 at r3](https://reviewable.io/reviews/cockroachdb/docs/9181#-MOcGgZH7cmgSvjrkmQm:-MOcGgZH7cmgSvjrkmQn:b-2dclgc) ([raw file](https://github.com/cockroachdb/docs/blob/39e1969e9c1134c0539d788e31276002ed6850a5/v20.2/import.md#L39)):*
> ```Markdown
> The source file URL _does_ require the `ADMIN` role in the following scenarios:
> 
> - S3 or GS using IMPLICIT credentials
> ```

same nit
___
*[v20.2/import-into.md, line 14 at r2](https://reviewable.io/reviews/cockroachdb/docs/9181#-MObXgtf0btWkog6xxxg-r2-14:-MOcHg36D9yVP3SItgkV:b-xfxfu0) ([raw file](https://github.com/cockroachdb/docs/blob/b0c7be2d2a564bfcf72bc53a40f7c59ae2c74166/v20.2/import-into.md#L14)):*
<details><summary><i>Previously, shermanCRL (Matt Sherman) wrote…</i></summary><blockquote>

I’m not sure what this line means, do we know what it’s meant to convey?
</blockquote></details>

I think it means that you can't use `IMPORT INTO ... WHERE ..` So if you had an existing row, you have to use `INSERT` to add data, rather than `IMPORT INTO`. 
___
*[v20.2/import-into.md, line 26 at r5](https://reviewable.io/reviews/cockroachdb/docs/9181#-MOcJPde2WasrA1ELph6:-MOcJPde2WasrA1ELph7:b4n63e3) ([raw file](https://github.com/cockroachdb/docs/blob/548e332f16509e23860d5565ebb3a883e7898a3e/v20.2/import-into.md#L26)):*
> ```Markdown
> The source file URL does _not_ require the `ADMIN` role in the following scenarios:
> 
> - S3 and GS using SPECIFIED (and not IMPLICIT) credentials. Azure is always SPECIFIED by default.
> ```

same nit about `SPECIFIED` / `IMPLICIT`
___
*[v20.2/import-into.md, line 31 at r5](https://reviewable.io/reviews/cockroachdb/docs/9181#-MOcJSwW4Qg5NWopJcB5:-MOcJSwW4Qg5NWopJcB6:b-2dclgc) ([raw file](https://github.com/cockroachdb/docs/blob/548e332f16509e23860d5565ebb3a883e7898a3e/v20.2/import-into.md#L31)):*
> ```Markdown
> The source file URL _does_ require the `ADMIN` role in the following scenarios:
> 
> - S3 or GS using IMPLICIT credentials
> ```

same nit

**Review by mwang1026 — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9181#-:-MOcRzl43lrQsgK3_Frs:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (and 1 stale) (waiting on @mwang1026 and @shermanCRL)

**Comment by shermanCRL** [comment]
___
*[v20.2/import-into.md, line 14 at r2](https://reviewable.io/reviews/cockroachdb/docs/9181#-MObXgtf0btWkog6xxxg-r2-14:-MOctX3z4RaSqRkmc964:b-g1inr3) ([raw file](https://github.com/cockroachdb/docs/blob/b0c7be2d2a564bfcf72bc53a40f7c59ae2c74166/v20.2/import-into.md#L14)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

I think it means that you can't use `IMPORT INTO ... WHERE ..` So if you had an existing row, you have to use `INSERT` to add data, rather than `IMPORT INTO`. 
</blockquote></details>

OK, I rephrased as: IMPORT INTO` does not offer `SELECT` or `WHERE` clauses to specify subsets of rows. To do this, use [`INSERT`](insert.html#insert-from-a-select-statement).

The other nits should be all set as well.

**Comment by shermanCRL** [comment]
___
*[v20.2/import-into.md, line 26 at r5](https://reviewable.io/reviews/cockroachdb/docs/9181#-MOcJPde2WasrA1ELph6:-MOcwAK9ALDOSL1ehLzL:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/548e332f16509e23860d5565ebb3a883e7898a3e/v20.2/import-into.md#L26)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

same nit about `SPECIFIED` / `IMPLICIT`
</blockquote></details>

Done.

**Pull request merged**